### PR TITLE
Us398/typography find help

### PR DESCRIPTION
**Pull request by leahtbriggs**

### CODE REVIEW

File: findHelpScreens/DealWithWorries.js [hunk 1]
```diff
@@ -187,6 +188,7 @@ const styles = StyleSheet.create({
         marginHorizontal: 18,
         textAlign: "left",
         alignSelf: "stretch",
+        fontFamily: "avenir-medium",
```
Comment (gcarvajal97) [inline findHelpScreens/DealWithWorries.js]: I thought avenir-medium was already set to be the default font?

**Review by gcarvajal97 — CHANGES_REQUESTED** [review]

File: findHelpScreens/HealthcareTeamScreens/AfterTheHospital.js [hunk 2]
```diff
@@ -52,12 +52,13 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/HealthcareTeamScreens/AfterTheHospital.js]: This is unnecessary

File: findHelpScreens/HealthcareTeamScreens/HealthcareTeam.js [hunk 3]
```diff
@@ -111,7 +111,7 @@ export default class HealthcareTeam extends Component {
                 <ScrollView contentContainerStyle={{ paddingTop: 0 }}>
                     <Card
                         image={healthTeam}
-                        featuredTitleStyle={{ fontSize: 40 }}
+                        featuredTitleStyle={{ fontSize: 40, fontFamily: 'avenir-medium'}}
```
Comment (ghost) [inline findHelpScreens/HealthcareTeamScreens/HealthcareTeam.js]: The title font isn't overridden on Android:
image

File: findHelpScreens/HealthcareTeamScreens/HealthcareTeam.js [hunk 4]
```diff
@@ -183,10 +183,12 @@ const styles = StyleSheet.create({
     },
     headerText: {
         fontSize: 16,
+        fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/HealthcareTeamScreens/HealthcareTeam.js]: This should be 'avenir-heavy' with no fontWeight

File: findHelpScreens/HealthcareTeamScreens/InTheED.js [hunk 5]
```diff
@@ -43,6 +43,7 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign:'left',
     fontSize: 14,
+    fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/HealthcareTeamScreens/InTheED.js]: This is unnecessary.

File: findHelpScreens/HealthcareTeamScreens/InTheHospital.js [hunk 6]
```diff
@@ -71,13 +71,14 @@ const styles = StyleSheet.create({
   },
   paragraph: {
     textAlign:'left',
+    fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/HealthcareTeamScreens/InTheHospital.js]: This is unnecessary.

File: findHelpScreens/HowToTalk.js [hunk 7]
```diff
@@ -121,11 +120,12 @@ const styles = StyleSheet.create({
         paddingLeft: 15,
         paddingRight:15,
         fontSize: 15,
+        fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/HowToTalk.js]: This is unnecessary.

File: findHelpScreens/HowToTalk.js [hunk 8]
```diff
@@ -139,14 +139,15 @@ const styles = StyleSheet.create({
...
-        fontWeight:'bold',
+        fontFamily: "avenir-heavy",
         fontSize:14
     },
     subBullet: {
       marginVertical:1,
       marginHorizontal:30,
       textAlign:'left',
       alignSelf:'stretch',
+      fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/HowToTalk.js]: This is unnecessary.

File: findHelpScreens/HowToTalk.js [hunk 9]
```diff
@@ -155,5 +156,6 @@ const styles = StyleSheet.create({
         paddingHorizontal:5,
         fontSize: 14,
         marginTop: 5,
+        fontFamily: "avenir-medium"
```
Comment (ghost) [inline findHelpScreens/HowToTalk.js]: This is unnecessary.

File: findHelpScreens/InjuryPainCareScreens/AdbominalTraumaScreen.js [hunk 10]
```diff
@@ -44,12 +44,13 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/AdbominalTraumaScreen.js]: This is unnecessary.

File: findHelpScreens/InjuryPainCareScreens/HeadInjuryScreen.js [hunk 11]
```diff
@@ -40,17 +40,19 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/HeadInjuryScreen.js]: This is unnecessary.

File: findHelpScreens/InjuryPainCareScreens/HeadInjuryScreen.js [hunk 12]
```diff
@@ -40,17 +40,19 @@ const styles = StyleSheet.create({
...
     textAlign: 'left',
     fontSize: 14,
-    fontWeight: 'bold',
+    fontFamily: 'avenir-heavy',
     marginVertical: 2,
     marginLeft: 10
   },
   link: {
     fontSize: 14,
+    fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/HeadInjuryScreen.js]: This is unnecessary.

File: findHelpScreens/HealthcareTeamScreens/HealthcareTeam.js (hunk 3 above)
Comment (ghost) [reply findHelpScreens/HealthcareTeamScreens/HealthcareTeam.js]: We're also missing fonts on the accordion:
image

File: findHelpScreens/HealthcareTeamScreens/HealthcareTeam.js [hunk 13]
```diff
@@ -213,10 +215,11 @@ const styles = StyleSheet.create({
         padding: 10
     },
     activeSelector: {
-        fontWeight: "bold"
+        fontFamily: 'avenir-heavy'
     },
     selectTitle: {
         fontSize: 14,
+        fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/HealthcareTeamScreens/HealthcareTeam.js]: This should be 'avenir-heavy' with no fontWeight property.

File: findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js [hunk 14]
```diff
@@ -209,12 +209,14 @@ const styles = StyleSheet.create({
     },
     headerText: {
         fontSize: 16,
-        fontWeight: "500"
+        fontWeight: "500",
+        fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js]: This should be 'avenir-heavy' with no fontWeight

File: findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js [hunk 15]
```diff
@@ -239,10 +241,11 @@ const styles = StyleSheet.create({
         padding: 10
     },
     activeSelector: {
-        fontWeight: "bold"
+        fontFamily: 'avenir-heavy'
     },
     selectTitle: {
         fontSize: 14,
+        fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js]: This should be 'avenir-heavy' with no fontWeight

File: findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js [hunk 16]
```diff
@@ -253,11 +256,12 @@ const styles = StyleSheet.create({
         paddingLeft: 15,
         paddingRight: 15,
         fontSize: 15,
+        fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js]: This 'subTitle' style is unused.

File: findHelpScreens/InjuryPainCareScreens/InjuryPreventionTipsScreen.js [hunk 17]
```diff
@@ -36,19 +36,21 @@ const styles = StyleSheet.create({
   },
   paragraph: {
     textAlign: 'left',
+    fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/InjuryPreventionTipsScreen.js]: This is unnecessary.

File: findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js [hunk 18]
```diff
@@ -131,7 +131,7 @@ export default class InjuryPainCare extends Component {
                 <ScrollView contentContainerStyle={{ paddingTop: 0 }}>
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/InjuryPainCare.js:131]: Card and accordion fonts aren't applied on Android:
image

File: findHelpScreens/InjuryPainCareScreens/InjuryPreventionTipsScreen.js [hunk 19]
```diff
@@ -36,19 +36,21 @@ const styles = StyleSheet.create({
...
     fontSize: 14,
-    fontWeight: 'bold',
+    fontFamily: 'avenir-heavy',
     marginVertical: 2,
     marginLeft: 10
   },
   link: {
     fontSize: 14,
     color: colors.accent,
+    fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/InjuryPreventionTipsScreen.js]: This is unnecessary.

File: findHelpScreens/InjuryPainCareScreens/ManagingPainScreen.js [hunk 20]
```diff
@@ -125,18 +125,20 @@ const styles = StyleSheet.create({
     paragraph: {
       textAlign: 'left',
       fontSize: 14,
+      fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/ManagingPainScreen.js]: This is unnecessary.

File: findHelpScreens/InjuryPainCareScreens/ManagingPainScreen.js [hunk 21]
```diff
@@ -125,18 +125,20 @@ const styles = StyleSheet.create({
...
-      fontWeight: 'bold',
+      fontFamily: 'avenir-heavy',
       marginVertical: 2,
     },
     paragraphUnderline: {
       textAlign: 'left',
       fontSize: 14,
       marginVertical: 2,
-      textDecorationLine: 'underline',
+      fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/ManagingPainScreen.js]: This is unnecessary.

File: findHelpScreens/InjuryPainCareScreens/TakingCareofCastScreen.js [hunk 22]
```diff
@@ -73,12 +73,13 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: 'avenir-medium',
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/TakingCareofCastScreen.js]: This is unnecessary.

File: findHelpScreens/InjuryPainCareScreens/UsingCrutchesScreen.js [hunk 23]
```diff
@@ -65,12 +65,13 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/InjuryPainCareScreens/UsingCrutchesScreen.js]: This is unnecessary.

File: findHelpScreens/QuickTipsScreens/QuickTips.js [hunk 24]
```diff
@@ -79,12 +79,14 @@ const styles = StyleSheet.create({
       color: colors.primary.white,
```
Comment (ghost) [inline findHelpScreens/QuickTipsScreens/QuickTips.js]: Looks like every card we have in Find Help is like this:
image

File: findHelpScreens/QuickTipsScreens/QuickTips.js [hunk 25]
```diff
@@ -107,6 +110,7 @@ const styles = StyleSheet.create({
       paddingVertical:1, 
       paddingHorizontal:5,
       fontSize: 14,
+      fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/QuickTipsScreens/QuickTips.js]: This is unnecessary.

File: findHelpScreens/outsideHelpScreens/HowToLook.js [hunk 26]
```diff
@@ -43,18 +43,20 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/outsideHelpScreens/HowToLook.js]: This is unnecessary.

File: findHelpScreens/outsideHelpScreens/HowToLook.js [hunk 27]
```diff
@@ -43,18 +43,20 @@ const styles = StyleSheet.create({
...
     fontSize: 14,
-    fontWeight: 'bold',
+    fontFamily: "avenir-heavy",
     marginVertical: 2,
     marginLeft: 10
   },
   link: {
     fontSize: 14,
     color: colors.accent,
+    fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/outsideHelpScreens/HowToLook.js]: This is unnecessary.

File: findHelpScreens/outsideHelpScreens/WhenAndHowToLookForYourself.js [hunk 28]
```diff
@@ -39,17 +39,19 @@ const styles = StyleSheet.create({
     textAlign: 'left',
     fontSize: 14,
     marginVertical: 2,
+    fontFamily: "avenir-medium"
```
Comment (ghost) [inline findHelpScreens/outsideHelpScreens/WhenAndHowToLookForYourself.js]: This is unnecessary.

File: findHelpScreens/outsideHelpScreens/WhenAndHowToLookForYourself.js [hunk 29]
```diff
@@ -39,17 +39,19 @@ const styles = StyleSheet.create({
...
-    fontWeight: 'bold',
+    fontFamily: "avenir-heavy",
     marginVertical: 2,
     marginLeft: 10
   },
   link: {
     fontSize: 14,
     color: colors.accent,
     textDecorationLine: 'underline',
+    fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/outsideHelpScreens/WhenAndHowToLookForYourself.js]: This is unnecessary.

File: findHelpScreens/outsideHelpScreens/WhenToGetOutsideHelp.js [hunk 30]
```diff
@@ -105,7 +105,7 @@ export default class WhenToGetOutsideHelpScreen extends Component {
                 <ScrollView contentContainerStyle={{ paddingTop: 0 }}>
                     <Card
```
Comment (ghost) [inline findHelpScreens/outsideHelpScreens/WhenToGetOutsideHelp.js:106]: image

File: findHelpScreens/outsideHelpScreens/WhenToGetOutsideHelp.js [hunk 31]
```diff
@@ -177,15 +177,17 @@ const styles = StyleSheet.create({
     },
     headerText: {
         fontSize: 16,
-        fontWeight: "500"
+        fontWeight: "500",
+        fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/outsideHelpScreens/WhenToGetOutsideHelp.js]: Should be 'avenir-heavy' with no fontWeight.

File: findHelpScreens/outsideHelpScreens/WhenToGetOutsideHelp.js [hunk 32]
```diff
@@ -177,15 +177,17 @@ const styles = StyleSheet.create({
...
     },
     paragraph: {
         alignSelf: "center",
         paddingVertical: 0,
         paddingHorizontal: 4,
         fontSize: 14,
         marginTop: 0,
-        marginBottom: 4
+        marginBottom: 4,
+        fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/outsideHelpScreens/WhenToGetOutsideHelp.js]: This is unnecessary.

File: findHelpScreens/outsideHelpScreens/WhenToGetOutsideHelp.js [hunk 33]
```diff
@@ -207,11 +209,12 @@ const styles = StyleSheet.create({
         padding: 10
     },
     activeSelector: {
-        fontWeight: "bold"
+        fontFamily: "avenir-heavy"
     },
     selectTitle: {
         fontSize: 14,
         fontWeight: "500",
+        fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/outsideHelpScreens/WhenToGetOutsideHelp.js]: This should be 'avenir-heavy' with no font weight.

File: findHelpScreens/outsideHelpScreens/WhenToLook.js [hunk 34]
```diff
@@ -40,18 +40,20 @@ const styles = StyleSheet.create({
   paragraph: {
     textAlign: 'left',
     fontSize: 14,
+    fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/outsideHelpScreens/WhenToLook.js]: This is unnecessary.

File: findHelpScreens/outsideHelpScreens/WhenToLook.js [hunk 35]
```diff
@@ -40,18 +40,20 @@ const styles = StyleSheet.create({
...
     fontSize: 14,
-    fontWeight: 'bold',
+    fontFamily: "avenir-heavy",
     marginVertical: 2,
     marginLeft: 10
   },
   link: {
     fontSize: 14,
     color: colors.accent,
+    fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/outsideHelpScreens/WhenToLook.js]: This is unnecessary.

File: findHelpScreens/selfCareScreens/AnxiousJumpy.js [hunk 36]
```diff
@@ -58,12 +58,13 @@ const styles = StyleSheet.create({
     paragraph: {
       textAlign: 'left',
       fontSize: 14,
+      fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/selfCareScreens/AnxiousJumpy.js]: This is unnecessary.

File: findHelpScreens/selfCareScreens/Avoidance.js [hunk 37]
```diff
@@ -47,13 +47,14 @@ const styles = StyleSheet.create({
     },
     paragraph: {
       textAlign: 'left',
+      fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/selfCareScreens/Avoidance.js]: This is unnecessary.

File: findHelpScreens/selfCareScreens/RelivingWhat.js [hunk 38]
```diff
@@ -42,13 +42,14 @@ const styles = StyleSheet.create({
     },
     paragraph: {
       textAlign: 'left',
+      fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/selfCareScreens/RelivingWhat.js]: This is unnecessary.

File: findHelpScreens/selfCareScreens/SelfCare.js [hunk 39]
```diff
@@ -192,13 +192,15 @@ const styles = StyleSheet.create({
     },
     headerText: {
         fontSize: 16,
-        fontWeight: "500"
+        fontWeight: "500",
+        fontFamily: "avenir-medium"
```
Comment (ghost) [inline findHelpScreens/selfCareScreens/SelfCare.js]: This should be 'avenir-heavy' with no fontWeight.

File: findHelpScreens/selfCareScreens/SelfCare.js [hunk 40]
```diff
@@ -192,13 +192,15 @@ const styles = StyleSheet.create({
...
     },
     paragraph: {
         alignSelf: "center",
         paddingVertical: 0,
         paddingHorizontal: 4,
         fontSize: 14,
+        fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/selfCareScreens/SelfCare.js]: This is unnecessary.

File: findHelpScreens/selfCareScreens/SelfCare.js [hunk 41]
```diff
@@ -222,11 +224,12 @@ const styles = StyleSheet.create({
         padding: 10
     },
     activeSelector: {
-        fontWeight: "bold"
+        fontFamily: "avenir-heavy"
     },
     selectTitle: {
         fontSize: 14,
         fontWeight: "500",
+        fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/selfCareScreens/SelfCare.js]: This should be 'avenir-heavy' with no fontWeight.

File: findHelpScreens/selfCareScreens/SelfCare.js [hunk 42]
```diff
@@ -236,11 +239,12 @@ const styles = StyleSheet.create({
         paddingLeft: 15,
         paddingRight: 15,
         fontSize: 15,
+        fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/selfCareScreens/SelfCare.js]: This is unnecessary.

File: findHelpScreens/selfCareScreens/SelfCareCard.js [hunk 43]
```diff
@@ -12,7 +12,7 @@ export default class SelfCareCard extends Component {
         return(
         <Card
```
Comment (ghost) [inline findHelpScreens/selfCareScreens/SelfCareCard.js:13]: image

File: findHelpScreens/selfCareScreens/SelfCareParents.js [hunk 44]
```diff
@@ -45,12 +45,13 @@ const styles = StyleSheet.create({
     paragraph: {
       textAlign: 'left',
       fontSize: 14,
+      fontFamily: "avenir-medium",
```
Comment (ghost) [inline findHelpScreens/selfCareScreens/SelfCareParents.js]: This is unnecessary.

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request merged**